api/blueprints/drbUtils.py

```diff
@@ -51,30 +57,31 @@ def totalCounts():
 @cross_origin(origins=os.environ.get('API_PROXY_CORS_ALLOWED', '*'))
 def getProxyResponse():
     proxyUrl = request.args.get('proxy_url')
-
     cleanUrl = unquote_plus(proxyUrl)
-
     urlParts = urlparse(cleanUrl)
 
+    logger.info(f'Received {request.method} request to proxy url: {proxyUrl}')
+
     while True:
         headResp = requests.head(
             cleanUrl, headers={'User-agent': 'Mozilla/5.0'}
         )
 
         statusCode = headResp.status_code
-        print(statusCode, cleanUrl)
+        
+        logger.info(f'HEAD response from {cleanUrl} returned status code {statusCode}')
+
         if statusCode in [200, 204]:
             break
         elif statusCode in [301, 302, 303, 307, 308]:
-            print(headResp.headers)
```

Comment:
Do the headers not need to be logged, or are they captured somewhere else?

Reply:
Also the print statement logs are hard to find in new relic because they don't have any of the other log metadata :(